### ISSUE TITLE
Security: array formulas are not escaped

### ISSUE THREAD

**Issue by tagliala** [caxlsx/caxlsx#192]
```rb
# frozen_string_literal: true

rows = [
  "{=cmd|'/C calc'!A0}"
].freeze

wb = xlsx_package.workbook

wb.add_worksheet(name: 'Test') do |sheet|
  sheet.add_row(rows, escape_formulas: true)
end
```

<img width="372" alt="image" src="https://user-images.githubusercontent.com/556268/229342114-623835bb-600a-4760-9be1-09ced465df4b.png">

Will be fixed by #189, but I suggest to fix this in a separate PR

**Comment by tagliala** [comment]
@johnnyshields any chance to just fix this (and add a spec)?

@kiskoza thanks for all the merges!

**Comment by tagliala** [comment]
Fixed in master via #186
